manifests/d/DrewNaylor/guinget/0.2.0.3/DrewNaylor.guinget.yaml

```
@@ -0,0 +1,31 @@
+PackageIdentifier: DrewNaylor.guinget
+Publisher: DrewNaylor
+PackageName: guinget
+PackageVersion: 0.2.0.3
+License: Apache License 2.0
+InstallerType: exe
+LicenseUrl: https://github.com/DrewNaylor/guinget/blob/master/LICENSE.txt
+Moniker: guinget
+Tags:
+- winget
+- package
+- gui
+- frontend
+- utility
+- tool
+- synaptic
+- package manager
+ShortDescription: DrewNaylor.guinget
+Description: Unofficial GUI for Microsoft's Windows Package Manager (winget). Kinda like Synaptic, but for Windows. Not associated with either Microsoft or the Synaptic project, and Microsoft does not endorse this software. You may need to run winget as an administrator to install this package due to issues where Inno Setup doesn't properly ask for UAC when installing with winget. Not associated with Inno Setup.
+PackageUrl: https://github.com/DrewNaylor/guinget/
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/DrewNaylor/guinget/releases/download/v0.2.0.3-alpha/guinget_v0.2.0.3-alpha.exe
+  InstallerSha256: D3A6E8C28CE684643AC8B511B853DBC721BBC88162479B80368BE908B7BB99BA
+  InstallerSwitches:
+    Silent: /VERYSILENT /ALLUSERS
+    SilentWithProgress: /SILENT /ALLUSERS
```

```suggestion
- Architecture: x86
  Scope: machine
  InstallerType: inno
  InstallerUrl: https://github.com/DrewNaylor/guinget/releases/download/v0.2.0.3-alpha/guinget_v0.2.0.3-alpha.exe
  InstallerSha256: D3A6E8C28CE684643AC8B511B853DBC721BBC88162479B80368BE908B7BB99BA
  InstallerSwitches:
    Custom: /ALLUSERS
- Architecture: x86
  Scope: user
  InstallerType: inno
  InstallerUrl: https://github.com/DrewNaylor/guinget/releases/download/v0.2.0.3-alpha/guinget_v0.2.0.3-alpha.exe
  InstallerSha256: D3A6E8C28CE684643AC8B511B853DBC721BBC88162479B80368BE908B7BB99BA
  InstallerSwitches:
    Custom: /CURRENTUSER
```
Cleanup and compatibility

---

I don't know if getting rid of the silent switches will allow it to continue to work with -s, but may as well since I haven't been looking at Inno-related stuff with regard to winget recently.

---

You basically only have to add silent switches to EXE's, and /NORESTART for safety